isis/src/base/apps/mappt/mappt.xml

```diff
@@ -308,6 +295,74 @@
         </description>
       </parameter>
 
+      <parameter name = "USECOORDLIST">
+        <type>boolean</type>
+        <default><item>FALSE</item></default>
+        <brief>
+          Enable coordinate list parameters in the GUI
+        </brief>

```

Comment:
Using this because it was also in campt, but it seems redundant if coordlist is also passed in? In the same spirit I didn't include coordtype as it also seemed redundant (campt has type for passed in points and coordtype for coordlist points) so it seems I should remove this as well. Although, keeping the extra params makes it more consistent with campt. Would like some input here.

Reply:
On a purely practical level, since checking USECOORDLIST in the GUI turns off the option for the user to enter a TYPE, COORDTYPE would need to be included so that a user could still enter this information.

Reply:
I know @scsides wrote something up on a former ticket explaining why ISIS handles things this way, but I'm having trouble finding it right now.

Reply:
Here it is! https://github.com/USGS-Astrogeology/ISIS3/issues/3189
(thanks @scsides)

Reply:
@kberryUSGS  Yeah, Stuart sent me the link ealier, it seems to be okay to leave as is.